ndbench-cass-plugins/src/main/java/com/netflix/ndbench/plugin/cass/CassJavaDriverPlugin.java

```diff
@@ -17,8 +17,6 @@
 package com.netflix.ndbench.plugin.cass;
 
 import com.datastax.driver.core.*;
```

Comment:
I am not sure what your convention for documenting the feature in the PR would be.  If you send me some examples i will be happy to follow...  For now, I just addressed the wiki documentation requirement  (sent you both email updates for documentation on wiki)